packages/preset-mini/src/_theme/types.ts

```diff
@@ -9,7 +9,7 @@ export interface ThemeAnimation {
 }
 
 export interface Colors {
-  [key: string]: Colors | string
+  [key: string]: Colors & { DEFAULT?: string } | string
```

Comment:
```suggestion
  DEFAULT?: string
  [key: string]: Colors | string
```

This would do the same, right?

Reply:
A little bit different.

### { [key: string]: Colors & { DEFAULT?: string } | string }
``` ts
colors: {
  red: {
     DEFAULT: ''
  },
  DEFAULT: '' // ts incorrect
}
```

###   { DEFAULT?: string; [key: string]: Colors | string }
``` ts
colors: {
  red: {
     DEFAULT: ''
  },
  DEFAULT: '' // ts correct
}
```